improver_tests/precipitation_type/freezing_rain/test_FreezingRain.py

```diff
@@ -63,10 +63,14 @@ def test_expected_result(input_cubes, expected_probabilities, expected_attribute
             assert result.name() == PROB_NAME.format(ACCUM_NAME)
             assert result.coord(var_name="threshold").name() == ACCUM_NAME
             assert result.coord(var_name="threshold").units == "mm"
+            assert result.cell_methods[0].method == "sum"
+            assert "time" in result.cell_methods[0].coord_names
         else:
             assert result.name() == PROB_NAME.format(RATE_NAME)
             assert result.coord(var_name="threshold").name() == RATE_NAME
             assert result.coord(var_name="threshold").units == "mm hr-1"
+            assert len(result.cell_methods) == 0
+            print(result.cell_methods)
```

Comment:
We don't generally include print statements in plugins/CLIs/tests. I don't think this will serve a purpose once merged into master and hence it can be removed.